packages/circuit-ui/styles/base.css

```diff
@@ -14,7 +14,7 @@
 
 html {
   box-sizing: border-box;
-  overflow-x: hidden;
+  overflow-x: clip;
```

Comment:
I learned recently that `overflow: hidden` prevents `position: sticky` from working as expected on any child elements. `overflow: clip` has the same visual effect, but doesn't interfere with `position: sticky`.